tests/tui/test_setup_wizard_view_coverage.py

```
@@ -0,0 +1,675 @@
+    mock_status_bar = MagicMock()
+    mock_app = MagicMock()
+    mock_app.query_one.return_value = mock_status_bar
+    
+    with patch.object(SetupWizardView, "app", new_callable=PropertyMock, return_value=mock_app):
+        view._set_status("success message")
+        mock_status_bar.set_status.assert_called_once_with("success message")
+
+
+def test_setup_wizard_view_run_hardware_detection_success() -> None:
+    """Verify background hardware detection updates status progress and stores capabilities."""
+    view, _, mock_app = _create_view_with_mocks()
```

Ruff (0.15.18)

[warning] Unpacked variable `mock_app` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)